com.unity.render-pipelines.core/Editor/PostProcess/LensFlareEditor.cs

```diff
@@ -0,0 +1,21 @@
+using System.IO;
+using UnityEditor;
+using UnityEngine;
+
+namespace UnityEngine
+{
+    /// <summary>
+    /// Editor for LensFlare (builtin): Editor to show an error message
+    /// </summary>
+    [CustomEditor(typeof(UnityEngine.Flare))]
+    public class LensFlareEditor : Editor
```

Comment:
And the editor can remain internal? We do not intend users to overwrite it?

Reply:
I don't know. @RSlysz ?

Reply:
Yes all Editor should be internal by default.